src/main/java/duke/Task.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.io.Serializable;
+public abstract class Task implements Serializable {
+
+    private String item;
+    private boolean isCompleted;
+
+    public Task(String item) {
+        this.item = item;
+        this.isCompleted = false;
+    }
+
+    /**
+     * Toggles the completed variable between true/false
+     */
+    public void completeToggle() {
```

Comment:
Would `toggleComplete` be a more suitable name, as it would start with the verb?